Jakefile

```diff
@@ -31,6 +31,15 @@ task('minify', function () {
   console.log('Minification completed.');
 });
 
+task('docPublish', function () {
+  exec('npm run doc -s');
```

Comment:
Why define a Jake task that then calls out to an NPM task? Let's not split up our build-logic across multiple systems. (This would be true whether we used Jake or Grunt or Gulp or some other.) In general NPM shortcuts should only be used to call out to other tasks during CI when there are built-in hook for these NPM scripts. (And they should probably point to proper build-system (Jake) tasks.)